src/api/lib/activexml/transport.rb

```diff
@@ -440,7 +440,7 @@ def handle_response(http_response)
 
     private
 
-    def symbolified_model(model_class)
+    def symbolize_model(model_class)
```

Comment:
This should be `symbolized_model` instead of `symbolize_model`

Reply:
Travis also realised 🤣